src/game/client/videoservices/OpusVorbisDecoder.hpp

```diff
@@ -0,0 +1,67 @@
+/*
+	MIT License
```

Comment:
should add this (and any other copyright notice that this PR adds) to `reactivedrop/thirdpartylegalnotices.txt`